### ISSUE TITLE
IE 11 is freezing after verifying Google Recaptcha2 audio challenge

### ISSUE THREAD

**Issue by nileshpenta** [nvaccess/nvda#7037]
Hi,

Looks like Google Recaptcha2 + IE 11 + NVDA Screen Reader combination is not working. As soon as we click on "Verify" after entering the audio numbers, IE 11 is freezing. Noticed the same in IE 10. I have checked the network log and confirm that the response from google recaptcha is returning 200 OK with valid response. The checkbox is not getting checked after this and webpage is freezing.

The below combinations do work.

Google Recaptcha2 + IE
Google Recaptcha2 + Chrome + NVDA Screen Reader 
Google Recaptcha2 + Firefox + NVDA Screen Reader
Google Recaptcha2 + Chrome
Google Recaptcha2 + Firefox

Please look into this.

Thanks,
Nilesh.

**Comment by nileshpenta** [comment]
Please use https://patrickhlauke.github.io/recaptcha/ to replicate the issue.

**Comment by PratikP1** [comment]
Do you have a test page or site for verification purposes?

**Comment by PratikP1** [comment]
I tested this sample multiple times with NVDA version next-13955,65ad3017.  I was unable to verify your findings. I used IE 11 on Windows 10 Pro Insider 15063

**Comment by nileshpenta** [comment]
Hi Pratik, thanks for checking. I am using Windows 7 Professional, IE 11.0.9600.18617 and NVDA 2017.1 (auto updates enabled)

Please check the below site https://www.tfwunlockpolicy.com/wps/portal/home/

**Comment by PratikP1** [comment]
I just tested the second example with the same configuration. It works as intended. I can’t verify the issue.

**Comment by Brian1Gaff** [comment]
You mean you can actually understand that speech?
 Blimey. My main thing with any site which employs capchars is to vote with 
my feet. Nothing but trouble, and most of my sighted friends agree.
 Google need to come up with something based on your human ness not dodgy 
audio and wobbly writing.

I have seen what you describe only yesterday, on a tell us what you think 
page on a web site. I got it to work in Firefox but after several tries at 
different 'challenges' their words not mine, I realised it must be speaking 
Klingon.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by Brian1Gaff** [comment]
It locked up on windows 7 and IE but not firefox.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by Brian1Gaff** [comment]
Hi again, the way it seems to be working in ie is to alt tab away, then back 
then get a new challenge, however sometimes it tells me it cannot process 
the request for some reason in speech, other times we get the klingon voice 
wityh hiss and long gaps, then I'm totally confused as the  fields I saw 
before have gone away.
 What a crazy system.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by puru0019** [comment]
Even i am facing the same issue same as @nileshpenta

**Comment by feerrenrut** [comment]
I couldn't reproduce the problem using IE 11 and NVDA Version: `next-14617,ef52ddc4` with the https://patrickhlauke.github.io/recaptcha/ site. 

It could be a problem with a specific version of the capture, the only version identifier I could find was on the URL of the recapture Javascript: `https://www.gstatic.com/recaptcha/api2/r20171109115411/recaptcha__en.js`
Indicating it is:
- api version 2
- and perhaps `r20171109115411` means released 2017 Nov 09 - 11:54:11 This is assuming American style dates.

**Comment by puru0019** [comment]
I am still facing the same issue in IE11 with NVDA 2017.3 and https://www.gstatic.com/recaptcha/api2/r20171109115411/recaptcha__en.js

**Comment by feerrenrut** [comment]
I still can't reproduce the IE freeze. @puru0019 could you attach an NVDA log file (with debug logging set) to this issue?

**Comment by JulienNado** [comment]
Hi,
We are facing the same issue with IE11 not responding after completing the recaptcha challenge when NVDA is on.

**Comment by JulienNado** [comment]
You can test on this page the Invisible reCaptcha making IE11 to freeze when NVDA is on : 
https://www.google.com/recaptcha/api2/demo?invisible=true

**Comment by ehollig** [comment]
@JulienNado are you using Windows 7? I am not able to reproduce using IE 11 on Windows 10 or Windows 8.1. It is possible this could be Windows 7 specific.

**Comment by JulienNado** [comment]
Yes, I'm using Windows 7.

**Comment by Adriani90** [comment]
@nileshpenta, @JulienNado, @puru0019 are you still able to reproduce this issue in NVDA 2019.1? Unfortunately when i try to test this, I get the message that my network might send automatic requests and I am adviced to try it later. This is not the case when I do it in Firefox or Chrome.

**Comment by Adriani90** [comment]
If you are still able to reproduce it, which addons do you have installed in your internet explorer?

**Comment by Adriani90** [comment]
@lukaszgo1 could you please try to reproduce this issue on Windows 7? It seems people on other windows versions cannot reproduce but only on Win7.

**Comment by Adriani90** [comment]
Since support for IE and for Windows 7 ended, I am closing this as abandoned. I don't expect any updates for better functionality in Windows 7 and IE setting.
